homework-2/src/app/page.tsx

```diff
@@ -1,10 +1,15 @@
+'use client'
+
+import { useRouter } from "next/navigation";
 import styles from "./page.module.css";
-import { ShowContainer } from "@/components/features/shows/ShowContainer/ShowContainer";
 
 export default function Home() {
+  const router = useRouter();
+
+  router.push('/all-shows');
```

Comment:
Staviri unutar useEffect hooka
```suggestion
  useEffect(() => {
    router.push('/all-shows');
  }, []);
```